contrib/test_jenkins.sh

```diff
@@ -24,8 +24,8 @@
 
 WORKSPACE=${WORKSPACE:=$PWD}
 ucx_inst=${WORKSPACE}/install
-CUDA_MODULE="dev/cuda10.1"
-GDRCOPY_MODULE="dev/gdrcopy"
+CUDA_MODULE="dev/cuda-latest"
+GDRCOPY_MODULE="dev/gdrcopy1.3_cuda10.1"
```

Comment:
cuda is latest but gdr is specific version.. should have gdr-latest as well..

Reply:
RoCM gdr copy build is failing with the latest gdr copy. (https://github.com/openucx/ucx/pull/4488#issuecomment-557395719).  I opened an issue for it (https://github.com/openucx/ucx/issues/4489).  we need some clarification from AMD. 
I would like go with this specific version to enable the cuda testing. cuda tests are disabled after cuda module names changed in the environment. Would like to merge this asap  enable the cuda testing again.

Reply:
@bureddy i've pushed commit to this PR to load Cuda by specific version as well, for consistency
we can change both to -latest when RoCM issue is fixed